Orange/data/io.py

```diff
@@ -300,7 +300,13 @@ class ExcelReader(_BaseExcelReader):
     @property
     def workbook(self) -> openpyxl.Workbook:
         if not self._workbook:
-            self._workbook = openpyxl.load_workbook(self.filename,
+            with warnings.catch_warnings():
+                # We don't care about extensions, but we hate warnings
```

Comment:
These are not file extensions but extensions to .xlsx format. Some internal MS stuff.